spec/integration/auth/sign_in_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe "Signing user in", type: :request do
```

Comment:
Try to use double quotes just if you need to interpolate something or if you have an apostrophe

Reply:
Roger that! 👌